bridge.py

```diff
@@ -3,32 +3,86 @@
 """
 from game_objects import Deck, Hand
 from score import Contract
+import numpy as np
 
-def run_game():
+def deal_game():
```

Comment:
Trying to run a main method now fails, since you updated the runner function. How are you running this? In a notebook?